datadragon/data_dragon.go

```diff
@@ -124,7 +124,7 @@ func (c *Client) GetChampionByID(id string) (ChampionDataExtended, error) {
 		return ChampionDataExtended{}, err
 	}
 	for _, champion := range champions {
-		if champion.ID == id {
+		if champion.Key == id {
```

Comment:
A few of these mappings were incorrect (maybe the values the riot API uses now changed)

Reply:
Could you expand on this a little bit more? Why do you think we should now use the key instead of the ID?

Reply:
Looking at [the data here](https://ddragon.leagueoflegends.com/cdn/11.14.1/data/en_US/champion.json), yo ucan see that Key is an int (a string that's an int..) and ID is a string (BASICALLY the champion name). The return type of ChampionID in the matches v5 endpoint ,AND the championIDS from /lol/platform/v3/champion-rotations are integers. Therefore it's not possible for them to map to ID in that data dragon json (i wonder if they changed up the json? i haven't touched ddragon or riot api in a few years..)